Formula/age.rb

```diff
@@ -0,0 +1,22 @@
+class Age < Formula
+  desc "Simple, modern, secure file encryption"
+  homepage "https://filippo.io/age"
+  url "https://github.com/FiloSottile/age/archive/v1.0.0.tar.gz"
+  sha256 "8d27684f62f9dc74014035e31619e2e07f8b56257b1075560456cbf05ddbcfce"
+  license "BSD-3-Clause"
+  head "https://github.com/FiloSottile/age.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-X main.Version=v#{version}"), "-o", bin, "filippo.io/age/cmd/..."
```

Comment:
```suggestion
    bin.mkpath
    system "go", "build", *std_go_args(ldflags: "-X main.Version=v#{version}"), "-o", bin, "filippo.io/age/cmd/..."
```
I think? Otherwise we can revert to the earlier version, but I think it would suffer from the same error.

Reply:
GitHub really needs a `git commit --amend` option for these review suggestions :wink:

Reply:
The GitHub UI does leave plenty to be desired.